clients/thymioupgrader/thymioupgrader.qrc

```diff
@@ -1,10 +1,11 @@
-<!DOCTYPE RCC><RCC version="1.0">
-<qresource>
-	<file>thymioupgrader_fr.qm</file>
-	<file>thymioupgrader_de.qm</file>
-	<file>thymioupgrader_es.qm</file>
-	<file>thymioupgrader_it.qm</file>
-	<file>images/fileopen.svgz</file>
-	<file>images/firmwareupgrade.png</file>
-</qresource>
+<RCC>
+    <qresource prefix="/">
+        <file>thymioupgrader_fr.qm</file>
+        <file>thymioupgrader_de.qm</file>
+        <file>thymioupgrader_es.qm</file>
+        <file>thymioupgrader_it.qm</file>
+        <file>images/fileopen.svgz</file>
+        <file>images/firmwareupgrade.png</file>
+        <file>images/thymioupgrader.svgz</file>
```

Comment:
please just add this line preserving indentation instead of changing the whole file